### DISCUSSION TITLE
Disable password expiration for certain users

### DISCUSSION THREAD

**Discussion by seek78** [dnnsoftware/Dnn.Platform#5720] [Q&A]
HI,
is there a way to disable password expiration for certain users?

Thank you
Seek

**Comment by Timo-Breumelhof** [comment]
No, by default there isn't.
Unless you would create your own Authentication provider I guess

**Reply by seek78** [reply]
Is it possible to make an implementation request?

**Reply by valadas** [reply]
You can find DNN Agencies here if you want to hire one to implement this for you. However I am not sure if that is a DNN thing or an asp.net authentication thing, soo would probably need a bit of investigative work on how that works before even starting to code a provider.

**Reply by seek78** [reply]
Thanks @valadas, but I can develop by myself.
Mine was a general request for the standard implementation of this feature.

**Reply by valadas** [reply]
I thought that's what you meant...

What do you mean by "Implementation Request"

**Reply by seek78** [reply]
> I thought that's what you meant...
> 
> What do you mean by "Implementation Request"

As standard DNN

**Comment by valadas** [comment]
Like I said I am not sure if DNN or AspNetSqlMembershipProvider handles password expiration, so step 1 would be to find where that is and potentially put a breakpoint and try to understand how that works. If it is PasswordResetExpiration here https://github.com/dnnsoftware/Dnn.Platform/blob/5def7cc2e7931bb1041b21540bde99f96874a5a9/DNN%20Platform/Library/Security/Membership/AspNetMembershipProvider.cs#L971 one could find all references in VisualStudio to know if that is it and find how it is used. 

Then potentially create a different provider. Or if your use case is to do it for all admins/host only, I could see that being a nice feature to have contributed here as I think many people would like that level of control to enforce that for admins but not disturb normal users.
